testcases/change_pasword/verify_change_pwd_error_message.robot

```diff
@@ -1,24 +1,26 @@
 *** Settings ***
-Library          DataDriver    file=../resources/change_pwd_fail_data.csv
-Resource         ../resources/import.resource
-Variables        ../resources/account_data.yaml
+Library    DataDriver    file=../../test_data/change_pwd_fail_data.csv
 
-Test Template    Invalid change password Scenarios
+Resource    ../../browser/browser.resource
+Resource    ../../page_objects/header_navigation.resource
+Resource    ../../page_objects/my_account_page.resource
+Resource    ../../page_objects/change_password_page.resource
+Resource    ../../page_objects/login_page.resource
 
-Suite Setup       Reach My Account Page After Logged In
+Test Template    Invalid Change Password Scenarios
+
+Suite Setup    Register User Then Reach Change Password Page
 Suite Teardown    Close Browser Window
 
 *** Test Cases ***
-Verify change password fail with invalid cred    ${testcase_id}  ${old_pwd}  ${new_pwd}  ${confirm_pwd}  
+Verify error message when changing password with invalid cred    ${testcase_id}  ${old_pwd}  ${new_pwd}  ${confirm_pwd}  
 ...  ${summary_error_message}  ${old_pwd_error_message}  ${new_pwd_error_message}  ${confirm_pwd_error_message}
-
 *** Keywords ***
-Invalid change password Scenarios
+Invalid Change Password Scenarios
     [Arguments]    ${testcase_id}  ${old_pwd}  ${new_pwd}  ${confirm_pwd}  
     ...  ${summary_error_message}  ${old_pwd_error_message}  ${new_pwd_error_message}  ${confirm_pwd_error_message}
-
-    Click Change Password Link
-    Input Old And New Password Then Click Change Password Button    ${old_pwd}  ${new_pwd}  ${confirm_pwd}
+    Input Old And New Password Then Click Change Password Button  
+    ...    ${old_pwd}  ${new_pwd}  ${confirm_pwd}
     Run Keyword If    '${testcase_id}' == 'ChangePWD_10'  Run Keywords  
```

Comment:
Please do not use ChangePWD_10 for comparison. It does not have the intended meaning. You can base on the input of old password, new password and confirmation to do that

IF ${current_pwd}==${EMPTY}
    Run Keyword    Verify Error Message Of Old Password
IF ${new_pwd}==${EMPTY}
    Run Keyword    Verify Error Message Of New Password
IF ${confirmation_pwd}==${EMPTY}
    Run Keyword     Verify Error Message Of Confirm New Password
----

or you can base on summary test case like your way.  But please remember try to make it simple and shorter